autopush/main.py

```diff
@@ -467,7 +467,8 @@ def endpoint_main(sysargs=None, use_files=True):
 
     # Endpoint HTTP router
     site = cyclone.web.Application([
-        (r"/push/([^\/]+)", EndpointHandler, dict(ap_settings=settings)),
+        (r"/push/(v\d+)?\/?([^\/]+)", EndpointHandler,
```

Comment:
doh, yeah, look ahead is better for this so we don't accidentally eat ids that begin with "v\d..."